package.json

```diff
@@ -52,22 +52,40 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "echo 'placeholder. no tests implemented'"
+    "test": "stripes test karma"
   },
   "devDependencies": {
+    "@bigtest/convergence": "^0.10.0",
+    "@bigtest/interactor": "^0.7.2",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.0",
+    "@bigtest/react": "^0.1.2",
+    "@folio/stripes-cli": "^1.3.0",
+    "babel-core": "^6.26.3",
     "babel-eslint": "^8.0.3",
+    "babel-loader": "^7.0.0",
+    "babel-polyfill": "^6.26.0",
+    "chai": "^4.0.2",
+    "chai-jquery": "^2.0.0",
     "eslint": "^4.8.0",
```

Comment:
I noticed Jenkins didn't report any results for `yarn lint`. I think it's because `@folio/eslint-config-stripes` needs newer `babel-eslint` and `eslint`. I handled that in https://github.com/folio-org/ui-myprofile/pull/16